src/Dapr.Client/DaprClient.cs

```diff
@@ -365,5 +365,17 @@ public async Task<StateEntry<TValue>> GetStateEntryAsync<TValue>(string storeNam
             string key,
             IReadOnlyDictionary<string, string> metadata = default,
             CancellationToken cancellationToken = default);
+
+        /// <summary>
+        /// Gets all secret values that the application is allowed to access from the secret store.
+        /// </summary>
+        /// <param name="storeName">Secret store name.</param>
+        /// <param name="metadata">A key/value pair that may be consumed by the secret store. This depends on the secret store used.</param>
```

Comment:
```suggestion
        /// <param name="metadata">A collection of metadata key-value pairs that will be provided to the secret store. The valid metadata keys and values are determined by the type of secret store used.</param>
```

We did a polish pass on these docs recently, this is similar to the text we're going with for metadata paramters